ddtrace/internal/forksafe.py

```diff
@@ -16,7 +16,7 @@ def ddtrace_after_in_child():
     # type: () -> None
     global _registry
 
-    for hook in _registry:
+    for hook in list(_registry):
```

Comment:
🤔 why is this necessary?

Reply:
This makes a copy of `_registry` to avoid an infinite loop. In this case, the infinite loop occurs when one enables the runtime metrics `fork` is called. At that point the runtime metrics are enabled again by the hook, which adds `_restart` back in `_registry` before the loop is finished